index.html

```diff
@@ -6,10 +6,10 @@
 
 <div class="jumbotron jumbotron-fluid">
   <div class="container text-center align-content-center">
-      <h2>PyTennessee is back!</h2>
+      <h2>PyTennessee 2019</h2>
       <div class="hero-date-info">
-        <h3>February 9-10, 2019 in Nashville, TN</h3>
-        <h4>Come for the hot chicken, stay for the community.</h4>
```

Comment:
This tagline was a nod to something someone did for us last year. I'd like to keep this the way it is. The rest of the changes are good, though.